IPR Policy.md

```diff
@@ -200,6 +200,7 @@ Documents other than [Living Standards][Living Standard] and [Review Drafts][rev
 [contributor]: ./IPR%20Policy.md#contributor
 [cw-agreement]: https://participate.whatwg.org/agreement
 [review-draft]: ./Workstream%20Policy.md#review-draft
+[BSD 3-Clause License]: ./BSD%203-Clause%20License.md
```

Comment:
It seems like this is the only change to IPR Policy.md: adding the link to BSD 3-Clause. Is that really right? I thought we were going to add some prose to this document.

It seems like, at minimum, [7.1 Copyright license for WHATWG publications](https://whatwg.org/ipr-policy#71-copyright-license-for-whatwg-publications) would need to change.

Reply:
We had the section "Incorporation in Software" which I removed in since the new suggested notice doesn't say that anymore: https://github.com/whatwg/sg/pull/115/commits/874739e674cc882030743824b65ef404790a9e93#diff-d4a1637beb56517ead607f9e4d10d9d2L199

Leaving only this change makes no sense and you're absolutely right, that wording should be updated, and should have been all along IMHO.

I've pushed a change now with this suggested wording:
> WHATWG Living Standards and Review Drafts are published under Creative Commons "Attribution 4.0 International (CC BY 4.0)" and a BSD 3-Clause License. Which license to exercise is at the option of the licensee, per the publication copyright notices.

Reply:
Sounds good, but I think 7.1.2. Other documents might want a similar change? Less critical than for specs, but it makes things simpler if the default copyright license is the same everywhere.

Reply:
I've added "and a BSD 3-Clause License" to the 7.1.2. Other documents.

There is a curious disconnect (although not a contradiction) between https://whatwg.org/workstream-policy#notice-for-other-publications and https://whatwg.org/ipr-policy#712-other-documents.

The former says simply "Other Publications published by the WHATWG bear text to be determined by the Steering Group" but in the latter spells out license and "unless a different license (or license waiver) is specified by the Steering Group." I've not updated the workstream policy, so there is no copyright notice I can link to. Since the copyright notices are what gives people the right to pick one license or the other, I've not tried to add "Which license to exercise is at the option of the licensee" for Other documents.

(Back after a vacation week.)

Reply:
That disconnect might be worth a separate issue, probably does not have to be changed here.

Reply:
Saying "and" and then having a following sentence that turns the AND into an OR seems more stressful for readers than working the first sentence to use the word "or" between the licenses. That is, "Foo or Bar, at your option" or "Foo and Bar, at the recipient's option" seem much clearer.

That is, I suggest formulating this such that the English use of "and" and "or" matches the SPDX meaning of "AND" and "OR".

Reply:
@hsivonen was that comment meant to be in reference to the LICENSE change rather than this IPR POLICY.md change? (Trying to guess which wording you object to.)

Reply:
@othermaciej Indeed, the outbound language at https://github.com/whatwg/sg/pull/115/files#diff-d485eedd5b4f85f1d8d32af46e9e4dfbR279 is clear. I withdraw my previous comment in this thread. Sorry about the noise.

Reply:
This is all further clarified now, and we have a "License Sample For Source Code.txt" showing how to use parts of spec text in source code.